<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_clientes</class>
 <widget class="QMainWindow" name="tela_clientes">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>708</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>708</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clientes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/img/img/logo_sex.png</normaloff>:/img/img/logo_sex.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_dados_cliente">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>790</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="title">
     <string>Dados do Cliente</string>
    </property>
    <widget class="QLabel" name="lb_telefone">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>53</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Telefone:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_telefone">
     <property name="geometry">
      <rect>
       <x>644</x>
       <y>53</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_numero_cliente">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>23</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Cadastro:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_numero_cadastro">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>23</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_nome">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>23</y>
       <width>39</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nome">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>23</y>
       <width>410</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>410</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>410</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>60</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_cpf">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>53</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>CPF:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_cpf">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>53</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>14</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_rg">
     <property name="geometry">
      <rect>
       <x>644</x>
       <y>23</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>RG:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rg">
     <property name="geometry">
      <rect>
       <x>665</x>
       <y>23</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_email">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>53</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>E-mail:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_email">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>53</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>20</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>450</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="maxCount">
      <number>60</number>
     </property>
    </widget>
    <widget class="QLabel" name="lb_comentario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Comentário</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_comentario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>380</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_endereo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>210</y>
      <width>790</width>
      <height>208</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="title">
     <string>Endereço</string>
    </property>
    <widget class="QLineEdit" name="le_cep">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>23</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_cep">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>CEP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_rua">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>23</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Rua:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rua">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>23</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_numero">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>23</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Número:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_numero">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>23</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_bairro">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>53</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Bairro:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_bairro">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>53</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>40</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_cidade">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>53</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Cidade:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_cidade">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>53</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>40</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_estado">
     <property name="geometry">
      <rect>
       <x>555</x>
       <y>53</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Estado:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_estado">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>53</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ponto_refe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>78</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Ponto de referência:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_ponto_referencia">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>770</width>
       <height>98</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>420</y>
      <width>790</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Histórico de Compras</string>
    </property>
    <widget class="QTableView" name="tv_historico_compras">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>770</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="Barra_de_menu">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionEditar_Contato"/>
   <addaction name="actionAdicionar_Contato"/>
  </widget>
  <action name="actionEditar_Contato">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/editar_cliente.png</normaloff>:/img/img/editar_cliente.png</iconset>
   </property>
   <property name="text">
    <string>Editar Contato</string>
   </property>
  </action>
  <action name="actionAdicionar_Contato">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/adiciona_clientes.png</normaloff>:/img/img/adiciona_clientes.png</iconset>
   </property>
   <property name="text">
    <string>Adicionar Contato</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>le_numero_cadastro</tabstop>
  <tabstop>le_nome</tabstop>
  <tabstop>le_rg</tabstop>
  <tabstop>le_cpf</tabstop>
  <tabstop>cb_email</tabstop>
  <tabstop>cb_telefone</tabstop>
  <tabstop>te_comentario</tabstop>
  <tabstop>le_cep</tabstop>
  <tabstop>le_rua</tabstop>
  <tabstop>le_numero</tabstop>
  <tabstop>le_bairro</tabstop>
  <tabstop>le_cidade</tabstop>
  <tabstop>le_estado</tabstop>
  <tabstop>te_ponto_referencia</tabstop>
  <tabstop>tv_historico_compras</tabstop>
 </tabstops>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
